frontend/src/pages/invite/join.ts

```diff
@@ -0,0 +1,165 @@
+          </h1>
+          ${this.inviteInfo.render({
+            complete: (inviteInfo) =>
+              renderInviteMessage(inviteInfo, {
+                isExistingUser: false,
+                isOrgMember: this.isLoggedIn,
+              }),
+          })}
+        </header>
+
+        <div class="max-w-md flex-1">
+          <div class="md:rounded-lg md:border md:bg-white md:p-12 md:shadow-lg">
+            ${this.inviteInfo.render({
+              pending: () => html`
+                <div class="flex items-center justify-center text-2xl">
+                  <sl-spinner></sl-spinner>
+                </div>
+              `,
+              complete: (inviteInfo) =>
+                this.isLoggedIn && inviteInfo && inviteInfo.firstOrgAdmin
+                  ? html`
+                      <btrix-org-form
+                        .authState=${this.authState}
+                        .orgId=${inviteInfo.oid}
+                        name=${this.signUpOrgDefaults?.name ||
+                        inviteInfo.orgName ||
+                        ""}
+                        slug=${this.signUpOrgDefaults?.slug ||
+                        inviteInfo.orgSlug ||
+                        ""}
+                      ></btrix-org-form>
+                    `
+                  : html`
+                      <btrix-sign-up-form
+                        email=${this.email!}
+                        inviteToken=${this.token!}
+                        .inviteInfo=${inviteInfo || undefined}
+                        submitLabel=${inviteInfo?.firstOrgAdmin
+                          ? msg("Next")
+                          : msg("Create Account")}
+                        @success=${this.onSignUpSuccess}
+                        @authenticated=${this.onAuthenticated}
+                      ></btrix-sign-up-form>
+                    `,
+              error: (err) =>
+                html`<btrix-alert variant="danger">
+                  <div>${err instanceof Error ? err.message : err}</div>
+                  <a
+                    href=${ROUTES.home}
+                    @click=${this.navLink}
+                    class="mt-3 inline-block underline hover:no-underline"
+                  >
+                    ${msg("Go to home page")}
+                  </a>
+                </btrix-alert> `,
+            })}
+          </div>
+        </div>
+      </section>
+    `;
+  }
+
+  private async getInviteInfo({
+    token,
+    email,
+  }: {
+    token: string;
+    email: string;
+  }): Promise<UserOrgInviteInfo | void> {
+    const resp = await fetch(
+      `/api/users/invite/${token}?email=${encodeURIComponent(email)}`,
+    );
+
+    if (resp.status === 200) {
+      return (await resp.json()) as UserOrgInviteInfo;
+    } else if (resp.status === 404) {
+      throw new Error(
+        msg(
+          "This invite doesn't exist or has expired. Please ask the organization administrator to resend an invitation.",
```

Comment:
Could we give the user some instructions on how to do this?  Support email?

Reply:
Similar to [previous comment](https://github.com/webrecorder/browsertrix/pull/1870#discussion_r1655279776), this retains the current functionality--it looks like a new change because I renamed the file--we can re-address once we can differentiate the hosted instance.